core/src/main/java/org/apache/cxf/staxutils/StaxSource.java

```diff
@@ -142,7 +142,10 @@ protected void parse() throws SAXException {
                         if (nsUri == null) {
                             nsUri = "";
                         }
-                        contentHandler.startPrefixMapping(nsPrefix, nsUri);
+                        // see please "com.ctc.wstx.returnNullForDefaultNamespace" property
+                        if (nsPrefix != null) {
```

Comment:
I'm not entirely sure what the purpose of this startPrefixMapping() call is, but I figure it's better to always make the call but use an empty string instead of zero. Just like nsUri above.

Reply:
Thanks @ud1 , I think it makes perfect sense